src/gui/session.py

```diff
@@ -13,15 +13,19 @@ def __init__(self, iface):
         self.iface = iface
         self.action = None
         self.session = SammoSession()
+        self.toolBar = None
```

Comment:
The sammo toolbar will contain multiple buttons not related to the session concept. So I think it'd be best to create the toolbar in `Sammo`:

``` python
class Sammo:
    def __init__(self, iface):
        self.iface = iface
        self.toolbar = ...
        self.actionSession = SammoActionSession(self.toolbar, iface)
```

Reply:
BTW, I didn't noticed in the previous PR but I'd prefer to restrict the access to `iface` which is very permissive. In our case:

``` python
self.actionSession = SammoActionSession(self.mainWindow(), self.toolbar)
```

where the window is a `parent` parameter in the `SammoActionSession` constructor (it's the usual Qt terminology).